doc/api/buffer.md

```diff
@@ -404,14 +404,14 @@ to initialize a `Buffer` to zeroes.
 Example:
 
 ```js
-const buf = new Buffer(5);
+const buf = new Buffer(10);
```

Comment:
In the first GIF you can see a test attempt with the old code example. See how both output lines are the same in contravention of the doc prediction.

![buffer1](https://cloud.githubusercontent.com/assets/10393198/20635739/e88090ec-b369-11e6-9610-c788ce6b6174.gif)

In the second GIF you can see a test attempt with the new code example. See how both output lines are different as the doc predicts.

![buffer2](https://cloud.githubusercontent.com/assets/10393198/20635742/f5bf9442-b369-11e6-99ae-81af35839bb4.gif)